gui/esp.hpp

```diff
@@ -50,6 +50,17 @@ static ImVec2 WorldToScreen(Vector2 pos)
 	return value;
 }
 
+static Vector2 ScreenToWorld(Vector2 pos)
+{
+	auto mainCamera = Camera_get_main(nullptr);
+
+	// use screen to world point
+	Vector3 vec3 = { pos.x, pos.y, 0 };
+	vec3 = Camera_ScreenToWorldPoint(mainCamera, vec3, nullptr);
```

Comment:
is there a particular reason you are saving the screen positions into vec3 and then saving the world positions into vec3 ?